src/providers/DevicesProvider/docs/AudioInputProvider.stories.mdx

```diff
@@ -4,6 +4,11 @@
 
 The `AudioInputProvider` provides a list of the user's available `audio-input` devices, the currently selected device, and the error if device selection fails.
 
+NOTE: If user chooses the default device as the current audio input device in Chrome, and plugs in a new device during the meeting that becomes the new default device, 
```

Comment:
Builders might know what is 'default' device. So should we add some details about it?

Reply:
Added some expected behavior to the "default" device